### PR TITLE
Oil shamblers no longer cause runtimes when killed

### PR DESCRIPTION
**Pull request by Shadow-Quill**
## About The Pull Request

Fixes runtimes due to a race condition when oil shamblers died.
Also should fix failing CI.

### CODE REVIEW

File: modular_skyrat/modules/biohazard_blob/code/biohazard_blob_mobs.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	. = ..()
 	update_overlays()
 
-/mob/living/simple_animal/hostile/biohazard_blob/oil_shambler/Destroy()
+/mob/living/simple_animal/hostile/biohazard_blob/oil_shambler/death()
 	visible_message(span_warning("The [src] evaporates!"))
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/biohazard_blob/code/biohazard_blob_mobs.dm]: We don't need to have the `The` here, so might as well remove it now.
```suggestion
	visible_message(span_warning("[src] evaporates!"))
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Don't forget to follow the pull request template!

**Comment by Shadow-Quill** [comment]
> Don't forget to follow the pull request template!

This doesn't touch the Skyrat Roleplay Experience, the proof that it works is that CI on my pr is passing, and the changes are not player/admin facing, so a changelog isn't required ^^

**Review by GoldenAlpharex — APPROVED** [review]
Even better fix, nice!

**Review by tf-4 — APPROVED** [review]

**Review by lessthnthree — APPROVED** [review]

**Pull request merged**